aspnetcore/migration/mvc.md

```diff
@@ -3,18 +3,189 @@ title: Migrate from ASP.NET MVC to ASP.NET Core MVC
+* The `UseStaticFiles` extension method adds the static file handler `Microsoft.AspNetCore.StaticFiles`. The `UseStaticFiles` extension method must be called before `UseRouting`. For more information, see [Static files](xref:fundamentals/static-files).
+* The `UseRouting` extension method adds routing. For more information, see [Application Startup](xref:fundamentals/startup) and [Routing](xref:fundamentals/routing).
+
+This existing configuration includes what is needed to migrate the example ASP.NET MVC project.
+
+## Migrate Controllers and views
+
+In the ASP.NET Core project a new empty controller class and view class would be added to serve as placeholders using the same names as the controller and view classes in any ASP.NET MVC project you migrate from.
+
+The ASP.NET Core *WebApp1* project already includes a minimal example controller and view by the same name as the ASP.NET MVC project. So those will serve as placeholders for the ASP.NET MVC controller and views to be migrated from the ASP.NET MVC *WebApp1* project.  
+
+1. Copy the methods from the ASP.NET MVC `HomeController` to replace the new ASP.NET Core `HomeController` methods. There's no need to change the return type of the action methods. The ASP.NET MVC built-in template's controller action method return type is [ActionResult](https://msdn.microsoft.com/library/system.web.mvc.actionresult(v=vs.118).aspx); in ASP.NET Core MVC, the action methods return `IActionResult` instead. `ActionResult` implements `IActionResult`.
+1. In the ASP.NET Core project, right-click the **Views** > **Home** folder, select  **Add** > **Existing Item...**.
+1. In the **Add Existing Item dialog**, navigate to the ASP.NET MVC WebApp1 project > Views > Home directory.
+1. Select the *About.cshtml*, *Contact.cshtml*, and *Index.cshtml* Razor view files, then select **Add**, replacing the existing files.
+1. Run the ASP.NET Core app.
+
+See [Controllers](xref:mvc/controllers/actions) and [Views](xref:mvc/views/overview) for more information.
+
+## Test each method
+
+The layout file and styles have not been migrated yet, so the rendered views only contain the content in the view files.  Also, the layout file generated links for the `About` and `Contact` views will not be available yet.
+
+* Invoke the rendered views from the browser on the running ASP.NET core app by replacing the current port number with the port number used in the ASP.NET core project. For example: `https://localhost:44375/home/about`.
+
+See [ASP.NET Core Controllers](xref:mvc/controllers/actions) and [ASP.NET Core Views](xref:mvc/views/overview) for more information.
+
+## Migrate Static content
+
+In previous versions of ASP.NET MVC, static content was hosted from the root of the web project and was intermixed with server-side files. In ASP.NET Core, static content is hosted in the **wwwroot** folder.
```

Comment:
folders are never bold.  make it a link to https://docs.microsoft.com/en-us/aspnet/core/fundamentals/?view=aspnetcore-3.1&tabs=windows#content-root

Reply:
+1 to Rick's comment. I recommend italicizing folder names. Bolding is useful for UI element names (e.g., **Solution Explorer**).

Reply:
Using this doc as an example which (mostly) uses "directory", I think directory is a better route and will go that way + italics unless I hear otherwise:
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/static-files?view=aspnetcore-3.1#serve-static-files.

Reply:
I prefer to code fence code `wwwroot` because it's hard coded in code. Italicised words are subject to translation, although in this case very unlikely. ***Bold*** is always wrong for folders/files.